cmd/osnadmin/main_test.go

```diff
@@ -708,6 +708,7 @@ var _ = Describe("osnadmin", func() {
 					"--client-key", clientKey,
 				}
 				output, exit, err := executeForArgs(args)
+				By(output)
```

Comment:
Why is this needed?

Reply:
note, this is draft and I wanted to see output of the command :)

Reply:
ack - it can disappear in the final PR then, thx!